src/store.js

```diff
@@ -15,10 +15,31 @@ const apiConnector = axios.create({
   responseType: 'json',
 })
 
+const axiosMiddlewareOptions = {
```

Comment:
Connait pas ... et comprends a peine le principe (récupère en local le token)

Reply:
* **Axios** is the lib used to send request to the API.
* **Axios Middleware** is used by redux to create automatically the `XXX_SUCCESS` and `XXX_FAIL` actions according to a `XXX` action.
* **Axios Middleware Options** is a set of options for the axios middleware.

Here we define an interceptor for the request. It will catch the request before she was sent, update the header to include the user token (bearer format) and let her to continue it's road to the API (having the user token to check the authorization).

About the local fetching of the token, it's just the way to store the token : we keep the user token (with the refresh token) provided by the API here and we use it when we needed. It's like store the token in a JWT or in a cookie.